src/marqo/tensor_search/models/add_docs_objects.py

```diff
@@ -41,7 +41,11 @@ class AddDocsParamsConfig:
 
 
 class AddDocsBodyParams(BaseModel):
-    """The parameters of the body parameters of tensor_search_add_documents() function"""
+    """
+    Representation of the body parameters of the API add_or_replace_documents() function.
```

Comment:
Revamped the description for this class, as the old one was vague and confusing. Feedback would be appreciated on the clarity of the new docstring!

Reply:
I'd prefer keeping this class and `tensor_search.add_documents()` decoupled from the API layer. `tensor_search.add_documents()`'s logic should be independent of whatever API (including a non REST protocols, such as gRPC) we choose.

Reply:
But isn't that the point of this class? To my understanding, it's called `AddDocsBodyParams` because it represents the parameters of the post request body. It then internally gets converted to a `AddDocsParams`.